boards/nrf52840dongle/Makefile.features

```diff
@@ -0,0 +1,10 @@
+CPU_MODEL = nrf52840xxaa
+
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_uart
+FEATURES_PROVIDED += periph_usbdev
+
+# Various other features (if any)
+FEATURES_PROVIDED += radio_nrf802154
```

Comment:
```suggestion
FEATURES_PROVIDED += radio_nrf802154
FEATURES_PROVIDED += bootloader_nrfutil
```

Reply:
Fixed in 981c1411d4dff28848f4ce85d2306ec89b5396d5